scripts/create_addon

```diff
@@ -137,6 +137,13 @@ pack_addon() {
       mkdir -p $ADDON_INSTALL_DIR/resources
       cp $ADDON_BUILD/$PKG_ADDON_ID/resources/icon.png $ADDON_INSTALL_DIR/resources/icon.png
     fi
+
+    # workaround for kodi pvr addons
+    if [ -f $ADDON_BUILD/$PKG_ADDON_ID/icon.png ]; then
+      mkdir -p $ADDON_INSTALL_DIR/resources
+      cp $ADDON_BUILD/$PKG_ADDON_ID/icon.png $ADDON_INSTALL_DIR/resources/icon.png
```

Comment:
L144 should be `cp $ADDON_BUILD/$PKG_ADDON_ID/icon.png $ADDON_INSTALL_DIR/icon.png` else we are still copying files to a resources subfolder that isn't referenced in the addon.xml we extract from the addon package and concatenate into addons.xml.gz.